docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -294,7 +298,7 @@ quarkus.oidc.introspection-credentials.secret=introspection-user-secret
 [[oidc-request-filters]]
 ==== OIDC request filters
 
-You can filter OIDC requests made by Quarkus to the OIDC provider by registering one or more `OidcRequestFiler` implementations, which can update or add new request headers, as well as log requests.
+You can filter OIDC requests made by Quarkus to the OIDC provider by registering one or more `OidcRequestFiler` implementations, which can update or add new request headers and log requests.
```

Comment:
`and` makes logging somehow related to the process of adding new headers, but it can be an independent activity

Reply:
modified slightly to say "and can also log requests."